src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainPhoto = document.querySelector('#largeImg');
+const images = document.querySelectorAll('.list-item');
+
+images.forEach((el) => {
+  el.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const imageSrc = e.target.parentElement.href;
```

Comment:
Directly accessing 'e.target' assumes that the clicked element will be the image itself. If the structure changes or if there are child elements, this might not hold true. It would be safer to use 'currentTarget' or ensure 'target' is the expected element.